docs/topics/wasm/wasm-debugging.md

```diff
@@ -81,19 +84,26 @@ steps should also work.
 To debug a Kotlin/Wasm application:
 
 1. In the browser window of the application, access developer tools by right-clicking and selecting the **Inspect** action.
+Alternatively, you can use the **Fn + F12** shortcut.
```

Comment:
I believe it's just `F12`. In your case, you probably need to press `Fn` due to default to media buttons.